src/H5Zzfp.c

```diff
@@ -29,16 +29,16 @@ and calls to bitstream methods with 'B ' as in
 #undef B
 #endif
 
+#include "zfp.h"
```

Comment:
We need to keep this as it was because it effects how ZFP and H5Z-ZFP get built into Silo

Reply:
Sorry this file should not change. let me fix the PR